pyabc/sampler/redis_eps/util.py

```diff
@@ -1,5 +1,13 @@
 import signal
 import sys
+import cloudpickle as pickle
+from redis import StrictRedis
+
+from .cmd import ACTIVE_SET, idfy
+from .redis_logging import logger
+
+# lock for accessing the active set data structure
+ACTIVE_SET_LOCK = "active_set_lock"
```

Comment:
Why is this here and not in the cmd.py file?

Reply:
yeah, probably makes sense to define in cmd.py all redis string constants.